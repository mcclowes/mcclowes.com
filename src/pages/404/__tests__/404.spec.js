import NotFoundPage from "../index";
import React from "react";
import { shallow } from "enzyme";

describe("NotFoundPage", () => {
  it("renders component", () => {
    const wrapper = shallow(<NotFoundPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
