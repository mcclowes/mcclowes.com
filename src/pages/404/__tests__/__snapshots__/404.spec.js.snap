// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders component 1`] = `
<Layout>
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</Layout>
`;
